plugins/funind/glob_termops.mli

```diff
@@ -122,3 +122,10 @@ val zeta_normalize : Glob_term.glob_constr -> Glob_term.glob_constr
 
 
 val expand_as : glob_constr -> glob_constr
+
+
+(* [resolve_and_replace_implicits ?expected_type env sigma rt] solves implicits of [rt] w.r.t. [env] and [sigma] and then replace them by their solution 
+ *)
+val resolve_and_replace_implicits :
```

Comment:
@maximedenes, @ppedrot: I'm not sure I can be of good help. My general feeling is that working at the `glob_constr` level is more difficult that working at the level of `constr`, and I think we already talked about this @forestjulien and I a couple of years ago. Also, I'm not sure I understand well what the implicit arguments have to do here, since they are supposed to be already added in the `constr_expr` to `glob_constr` phase.